### ISSUE TITLE
issue with feature search

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1399]
I annotated 49 private  staph genomes.  I selected them in the genome browser.  Then I clicked the features tab.  I was startled when the the displayed features were not from my original selected set.  I should not have to create a genome group to do this search.

**Comment by chmao1** [comment]
@jimdavis1 Could you check this again and see whether this problem still exists?

**Comment by mshukla1** [comment]
@jimdavis1 without screenshot or URL, it is hard to follow the problem you mentioned. 

Can you provide some more details / screenshot to help understand the problem?
